heron/common/src/python/constants.py

```diff
@@ -25,6 +28,32 @@
 TOPOLOGY_STMGRS = "topology.stmgrs"
 TOPOLOGY_WORKER_CHILDOPTS = "topology.worker.childopts"
 TOPOLOGY_ADDITIONAL_CLASSPATH = "topology.additional.classpath"
+
+TOPOLOGY_TICK_TUPLE_FREQ_SECS = "topology.tick.tuple.freq.secs"
+TOPOLOGY_ACKING_ENABLED = "topology.acking"
+TOPOLOGY_MAX_SPOUT_PENDING = "topology.max.spout.pending"
+
 RAM_FOR_STMGR = 1 * GB
 DEFAULT_RAM_FOR_INSTANCE = 1 * GB
 DEFAULT_DISK_PADDING_PER_CONTAINER = 12 * GB
+
+LOGGING_DIRECTORY = 'heron.logging.directory'
+MAX_LOG_FILES = 'heron.logging.maximum.files'
+MAX_LOG_SIZE_MB = 'heron.logging.maximum.size.mb'
+
+INSTANCE_ACK_BATCH_TIME_MS = "heron.instance.ack.batch.time.ms"
```

Comment:
There were a bunch of other heron instance options. Wondering if they are not needed?